### PR TITLE
[_]:(hotfix) Fix incorrect sorting for drive items

### PR DESCRIPTION
**Pull request by PixoDev**
Some items were being sorted incorrectly in Drive.

What changed:

- The size being received was an string, so sorting by size was incorrect
- The name of the files has been converted to hex, so we can sort them in any language

### CODE REVIEW

**Comment by PixoDev** [comment]
https://github.com/internxt/drive-server/blob/master/src/app/models/file.ts#L41

As @JoanVicens  pointed the output from this can be an string, I verified that in mobile an string is received

File: src/screens/DriveScreen/index.tsx [hunk 1]
```diff
@@ -52,11 +52,8 @@ function DriveScreen({ navigation }: TabExplorerScreenProps<'Drive'>): JSX.Eleme
   const driveSortedItems = useMemo(
     () => [
       ...driveUploadingItems,
-      ...driveItems.sort(drive.file.getSortFunction(sortMode)).sort((a, b) => {
-        const aValue = a.data.fileId ? 1 : 0;
-        const bValue = b.data.fileId ? 1 : 0;
-        return aValue - bValue;
-      }),
+      ...driveItems.filter((item) => !item.data.fileId).sort(drive.file.getSortFunction(sortMode)),
```
Comment (sg-gs) [inline src/screens/DriveScreen/index.tsx:55]: use `concat`

Comment (sg-gs) [inline src/screens/DriveScreen/index.tsx:55]: use concat

**Review by sg-gs — CHANGES_REQUESTED** [review]

File: src/screens/DriveScreen/index.tsx (hunk 1 above)
Comment (PixoDev) [reply src/screens/DriveScreen/index.tsx]: Solved

**Pull request merged**